packages/icon/README.md

```diff
@@ -88,3 +88,34 @@ const myGlyphs = {
 
 const MyIconComponent = createIconComponent(myGlyphs);
 ```
+
+# Using individual Icons
+
+We export each icon individually to enable using them directly. This is particularly useful if you don't intend to use more than a few icons, and should allow bundlers to omit unused Icons more effectively, reducing bundle size.
+
+> **NOTE**
+>
+> This API is only useful if you don't use the default `Icon` component elsewhere in your bundled code. If you intend to use a large portion of the default set of icons, we recommend continuing to use the default `Icon` component for the best developer experience.
+
+## Example
+
+```js
+import { glyphs } from '@leafygreen-ui/icon';
```

Comment:
API should be `<Icon.Plus />`, no? Or `<Glyph.Plus />`. Plural implies a collection, this is an enumeration of items.